auslib/web/admin/views/rules.py

```diff
@@ -46,32 +47,37 @@ def get(self, **kwargs):
     def _post(self, transaction, changed_by):
         # a Post here creates a new rule
         form = RuleForm()
-        releaseNames = dbo.releases.getReleaseNames()
-        form.mapping.choices = [(item['name'], item['name']) for item in releaseNames]
+        release_names = dbo.releases.getReleaseNames()
+        form.mapping.choices = [(item['name'], item['name']) for item in release_names]
         form.mapping.choices.insert(0, ('', 'NULL'))
 
-        if not form.validate():
-            self.log.warning("Bad input: %s", form.errors)
-            return Response(status=400, response=json.dumps(form.errors))
-
-        what = dict(backgroundRate=form.backgroundRate.data,
-                    mapping=form.mapping.data,
-                    fallbackMapping=form.fallbackMapping.data,
-                    priority=form.priority.data,
-                    alias=form.alias.data,
-                    product=form.product.data,
-                    version=form.version.data,
-                    buildID=form.buildID.data,
-                    channel=form.channel.data,
-                    locale=form.locale.data,
-                    distribution=form.distribution.data,
-                    buildTarget=form.buildTarget.data,
-                    osVersion=form.osVersion.data,
-                    systemCapabilities=form.systemCapabilities.data,
-                    distVersion=form.distVersion.data,
-                    comment=form.comment.data,
-                    update_type=form.update_type.data,
-                    headerArchitecture=form.headerArchitecture.data)
+        # Replaces wtfForms validations
+        nullable_field_dict = dict()
+        for key in request.json:
+            if isinstance(request.json[key], str_types):
+                nullable_field_dict[key] = None if request.json[key].strip() == '' else request.json[key].strip()
```

Comment:
let me check whether wtfForms stripped white spaces or not. AFAIK empty string was passed as None in python or NULL in DB.